src/index.css

```diff
@@ -1,3 +1,277 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;300;400;500;600;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;300;500;700&display=swap');
+
 body {
   box-sizing: border-box;
+  padding: 4%;
+  margin: 0;
+  font-family: "Montserrat", sans-serif;
+  background-color: #f3f4f8;
+}
+
+#root {
+  background-color: #feffff;
+}
+
+header {
+  display: flex;
+  padding: 24px 4%;
+  justify-content: space-between;
+  border-bottom: 1px solid #e8e8e8;
+}
+
+nav {
+  display: flex;
+  gap: 40px;
+}
+
+nav a {
+  text-decoration: none;
+  display: flex;
+  align-items: center;
+}
+
+.header-title {
+  margin: 0;
+  font-size: 1.875rem;
+  font-weight: 700;
+  color: #0290ff;
+}
+
+.navbar-ul {
+  padding: 0;
+  list-style: none;
+  margin: 0;
+  display: flex;
+  gap: 40px;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav-link {
+  font-size: 0.813rem;
+  color: #12121250;
+  letter-spacing: 1.9px;
+  text-transform: uppercase;
+}
+
+.active {
+  color: #121212;
+}
+
+.app-container {
+  background-color: #fafafa;
+  padding: 4%;
+}
+
+.user-icon {
+  color: #0290ff;
+}
+
+.book-container {
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+  border-radius: 4px;
+  border: 1px solid #e8e8e8;
+  padding: 2rem;
+  margin-bottom: 2.5rem;
+}
+
+.book-category {
+  margin: 0;
+  font-size: 0.875rem;
+  color: #888;
+}
+
+.book-name {
+  margin: 0.188rem 0;
+  font-size: 1.375rem;
+  letter-spacing: -0.2px;
+  font-family: "Roboto Slab", serif;
+  font-weight: 700;
+  color: #121212;
+}
+
+.book-author {
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #4386bf;
+  margin: 0;
+  font-family: "Roboto Slab", serif;
+}
+
+.book-buttons-container {
+  display: flex;
+  align-items: center;
+  margin-top: 1.313rem;
+}
+
+.book-button {
+  background-color: transparent;
+  border: none;
+  padding: 0;
+  color: #4386bf;
+  font-size: 0.875rem;
+  font-weight: 300;
+}
+
+.book-button:nth-child(2) {
+  margin: 0 0.5rem;
+  cursor: pointer;
+  padding: 0.3rem 0.5rem;
+  border-left: 2px solid #e8e8e8;
+  border-right: 2px solid #e8e8e8;
+}
+
+.progress-container {
+  display: flex;
+  align-items: center;
+}
+
+.circular-progress-bar {
+  width: 5.625rem;
+  height: 5.625rem;
+  padding: 0.63rem 0.625rem 0.75rem 0.75rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.circular-progress {
+  border-radius: 50%;
+  width: 4.25rem;
+  height: 4.25rem;
+  border: 5px solid #307bbe;
+  border-left-color: #e8e8e8;
+  transform: rotate(45deg);
+}
+
+.complete-precent {
+  font-family: "Montserrat", sans-serif;
+  color: #121212;
+  font-size: 2rem;
+  margin: 0;
+  padding: 0;
+}
+
+.completed {
+  font-size: 0.875rem;
+  color: #12121250;
```

Comment:
- [ ] Kindly do the same for this color proterty by creating a reusable class for it as it is used more than once in this file

Reply:
Thanks for the suggestion. Fixed🛠️✅